sufia-models/app/jobs/resolrize_job.rb

```diff
@@ -4,13 +4,6 @@ def queue_name
   end
 
   def run
-    require 'active_fedora/version'
```

Comment:
resolrize_job was basically broken in its current form because it reindexed with an (F3-style) assumption that identifiers start with PIDs. This change makes reindexing work again under F4.